packages/core/src/Modules/ComplaintsPolicy/Components/complaints-policy-content.jsx

```diff
@@ -6,38 +6,18 @@ import { observer, useStore } from '@deriv/stores';
 
 const getIntroductionText = (landing_company_shortcode, mt5_login_list) => {
     // * mt5_login_list returns these:
-    // landing_company_short: "svg" | "malta" | "maltainvest" |  "vanuatu"  | "labuan" | "bvi"
+    // landing_company_short: "svg" | "maltainvest" |  "vanuatu"  | "labuan" | "bvi"
```

Comment:
Can we just remove this whole line as this is already commented 😬 ?

Reply:
This is just a comment to show the possible responses from the `landing_company_short`. I believe it is safe to keep it here as I am not refactoring the file, I simply removed some conditions.